### ISSUE TITLE
Opening file does not keep them open

### ISSUE THREAD

**Issue by foyarash** [Microsoft/vscode#69381]
- VSCode Version: 1.32.0-insider
- OS Version: MacOS 10.14.2

Steps to Reproduce:

1. Double click on a file on the file explorer
2. The file should be kept open, but the file name stays in italic, opening another file replaces the current opened file.

Does this issue occur when all extensions are disabled?: Yes

**Comment by rebornix** [comment]
@foyarash you can double click the tab to keep the file always open but it sounds like a feature request.

**Comment by yume-chan** [comment]
@rebornix double click in explorer used to change a preview editor to a normal editor, but stopped working in recent versions.

**Comment by foyarash** [comment]
@rebornix i'm talking about the normal feature we have on the stable vs code version, when i double click on a file on the file explorer, i expect it to be pinned, but on the insider version that's not the case

**Comment by Ash258** [comment]
Since 2 (or 3) insiders back, double clicked files no longer open and keep opened.

![Double clicked](https://i.imgur.com/Bxa6gIO.gif)

Also there is bug, when you close file, which is selected inside explorer, you cannot open it again until you deselect and select again (see first part of gif)

Version: 1.32.0-insider
Commit: 7d0c9e6fa0bedcf71bdae86c02611b65362795a3
Date: 2019-02-26T06:19:00.351Z
Electron: 3.1.3
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17134

**Comment by Ash258** [comment]
Should be duplicate of #69284

**Comment by foyarash** [comment]
Thanks @Ash258  for pointing out the first issue mentionning the problem, closing this issue.
